src/index.js

```diff
@@ -68,6 +68,9 @@ async function main() {
 		startNode(relay_chain_bin, name, wsPort, port, spec, flags);
 	}
 
+	// Wait a second for relay chan validator API endpoints to become available
```

Comment:
```suggestion
	// Wait for some time for the relay chain RPC endpoints to become available.
```

I am not really convinced that this is a real problem though.

Reply:
I ran into an issue where the `await connect` promise on the next line fails with a rejection and crashes the whole process without this.

Reply:
Oh, that's curious. Are you sure that the failure is related to the endpoint being not available? In my experience, it's would more likely to fall into an infinite loop, in case the node didn't start correctly, rather than error out.

Moreover, if that's indeed a problem, I believe that a delay is not an appropriate solution here.

Reply:
no, not sure the failure is due to that. i think the node did start correctly but it just took a while to come up perhaps? and yes, you're right there is likely a better solution that a delay

Reply:
yeah, usually it takes around 10 seconds (or rather 10 reconnection attempts) and after that it finally connects